systems-manager/Packaging-utilities/examples/linux-sensor-binary/packager.py

```diff
@@ -278,6 +279,10 @@ def _get_digest(self, files):
         print(
             "Skipping AWS upload: please provide --aws_region, --ssm_automation_doc_name, and --s3bucket command-line options for upload")
         print("Package has been built successfully.")
-    else:
+    elif region and s3bucket:
+        S3BucketUpdater(region).update(s3bucket, files)
+    elif region and s3bucket and package_name:
         S3BucketUpdater(region).update(s3bucket, files)
         SSMPackageUpdater(region).update(package_name, "./s3-bucket/manifest.json")
+    else:
+        print("Nothing to do ... specify region + bucket or region + bucket + package_name")
```

Comment:
How are these options received? Can an example be given?

eg ``specify a region with the --region <region> flag``